docs/src/components/fields/RadioGroupTextField.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { FormControl, FormGroup, ControlLabel, Col, ButtonToolbar, Radio } from 'react-bootstrap';
+import { IField, IFieldBuilderProps, IFieldInputProps, IFieldInputInjector, createFieldBuilderWrapper } from 'react-dynamic-formbuilder';
+
+interface IState {
+}
+
+export class RadioGroupTextField extends React.PureComponent<IFieldInputProps & IFieldBuilderProps, IState> implements IFieldInputInjector {
+    public static defaultProps = {
+        value: '',
+    } as IFieldInputProps & IFieldBuilderProps
+
+    constructor(props: IFieldInputProps & IFieldBuilderProps) {
+        super(props);
+    }
+
+    public render() {
+        const { label, fields } = this.props.field;
+        const { others } = this.props.field.options;
+        return (
+            <div>
+                <FormGroup className='clearfix'>
+                    <Col componentClass={ControlLabel} md={5}>{label}</Col>
+                    <Col md={7}>
+                        {fields.map((item)=>{
+                            //TODO: (xiongchao) radio name is wrong
+                            return <Radio name={item.type} key={item.id}>{item.label}</Radio>
+                        })}
+                        {others.checked &&
+                            <div>
+                                <Radio>{others.label}</Radio>
+                                <FormControl
+                                    type='text'
+                                    value={others.value}
```

Comment:
This cannot work. field.option is form template definition. It cannot be saved into each data, and it won't change unless someone use option editor to modify.

The value in the props is the only information which will be saved in the database. Value can be a jObject if necessary.